src/status_im/chat/styles/message/message.cljs

```diff
@@ -48,18 +48,18 @@
             :align-self     align
             :align-items    align})))
 
-(defn message-timestamp [justify-timestamp?]
-  (merge {:color      colors/gray
+(defn message-timestamp [justify-timestamp? outgoing]
+  (merge {:color      (if outgoing colors/wild-blue-yonder colors/gray)
           :font-size  10
           :align-self :flex-end
           :opacity    0.5}
          (when justify-timestamp? {:position :absolute
                                    :bottom   10
                                    :right    12})))
 
-(def message-timestamp-placeholder
-  (assoc (message-timestamp false)
-         :color styles/color-white))
+(defn message-timestamp-placeholder [outgoing]
+  (assoc (message-timestamp false outgoing)
+         :color (if outgoing colors/hawkes-blue styles/color-white)))
```

Comment:
Maybe there could be a more abstract fn used to compose message style? `color` is overridden here so passing `outgoing` to `message-timestamp` is void.

Reply:
@jeluard makes sense, refactored that.